serving/resource_model_serving.go

```diff
@@ -0,0 +1,75 @@
+package serving
+
+import (
+	"context"
+	"time"
+
+	"github.com/databricks/databricks-sdk-go/retries"
+	"github.com/databricks/databricks-sdk-go/service/endpoints"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+const DefaultProvisionTimeout = 45 * time.Minute
+
+func ResourceModelServing() *schema.Resource {
+	s := common.StructToSchema(
+		endpoints.CreateServingEndpoint{},
+		func(m map[string]*schema.Schema) map[string]*schema.Schema {
+			m["name"].ForceNew = true
```

Comment:
This means we'd create a new endpoint on name changes, is that right? Is that the intended behavior? Please document if so.

Reply:
This should be the intended behavior since it doesn't look like you can rename endpoints (ML Inference correct me if I'm wrong @sueann @mehtayogita). I can add a note to the `name` field in the README

Reply:
@arpitjasa-db yes, please add the line in the `model_serving.md`. I confirm what Arpit did here is logical for TF.